app/presets/aura/dialog/index.js

```
@@ -0,0 +1,160 @@
+
+			// Background and Effects
+			{ 'has-[.mask-active]:bg-transparent bg-black/40': props.modal },
+
+			'overflow-y-scroll',
+		],
+	}),
+	transition: ({ props }) => {
+		// Use default animation for the "position": "top"
+		// return props.position === 'top'
+		// eslint-disable-next-line no-constant-condition
+		return false
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)